_posts/en/2019-03-19-bech32-sending-support.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Bech32 Sending Support
+permalink: /en/bech32-sending-support/
+name: 2019-03-19-bech32-sending-support
+type: posts
+layout: post
+lang: en
+
+excerpt: >
+  Copies of all published parts from our weekly series on bech32 sending
+  support, scheduled to run 24 weeks from March 19th to August 28th,
```

Comment:
If you s/scheduled to run/running for/ we won't need to update this once the series is complete.